api/STYLE.md

```diff
@@ -130,9 +130,9 @@ To add an extension config to the API, the steps below should be followed:
    )
    ```
 1. Update [source/extensions/extensions_metadata.yaml](../source/extensions/extensions_metadata.yaml)
-   with the category, security posture, and status. The category field will have to match an
+   with the category, [security posture, and status](../EXTENSION_POLICY.md#extension-stability-and-security-posture). The category field will have to match an
    annotation of the form `// [#extension-category: your.extension.category]`
-   in one of the proto files for the docs build to pass.
+   in a message in one of the proto files for the docs build to pass. You may also need `// [#extension: your.extension.category.extension_name]`.
```

Comment:
this is a bit wordy and hard to follow 

are we saying something like...

> any extension category added to the metadata.yaml file should be present in one and only one proto file as an annotation (...)

and

> every configured extension category should have at least one extension defined. This is can be done in the relevant proto file with the annotation (...)


not sure if my suggestion is helpful or correct, but i think these sentences need breaking up and some clarification

Reply:
I've had a go at making this clearer (and in the process also clarified my own understanding of what these annotations mean!)
I'm not fully sure why extension-category must be in only one place, it seems like theoretically it should be possible to have more than one thing extended by the same category of extension, but I didn't find an example of that happening so I guess it might be an arbitrary constraint.

Now documented with codeblock examples so you can easily see where both annotations belong.